.github/workflows/generate-alpha-tag.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Gradle build
         uses: gradle/gradle-build-action@v2.2.0
         with:
-          arguments: --full-stacktrace assemble "-Psemver.stage=alpha"
+          arguments: --full-stacktrace build "-Psemver.stage=alpha"
```

Comment:
Here is it not necessary to inject it?

Reply:
We want to run the full build, with tests before we create the tag/release, right?
The publish workflow is completely separate from this, so they shouldn't interfere with each other.